<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PDFSetupDialog</class>
 <widget class="QDialog" name="PDFSetupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>883</width>
    <height>451</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Save pdf to:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="pathEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="browseButton">
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <property name="verticalSpacing">
          <number>15</number>
         </property>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Horizontal margin ratio</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSlider" name="hMarginSlider">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Horizontal spacing ratio</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Vertical margin ratio</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Vertical spacing ratio</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSlider" name="hSpacingSlider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QSlider" name="vMarginSlider">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QSlider" name="vSpacingSlider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Show numbers</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QCheckBox" name="numberBox">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Diagrams per row</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="rowSizeBox">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>12</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="preview">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>400</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>400</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::NoTextInteraction</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PDFSetupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PDFSetupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>browseButton</sender>
   <signal>clicked()</signal>
   <receiver>PDFSetupDialog</receiver>
   <slot>browsePath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>325</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hMarginSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>PDFSetupDialog</receiver>
   <slot>setHMarginRatio(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hSpacingSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>PDFSetupDialog</receiver>
   <slot>setHSpacingRatio(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vMarginSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>PDFSetupDialog</receiver>
   <slot>setVMarginRatio(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vSpacingSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>PDFSetupDialog</receiver>
   <slot>setVSpacingRatio(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>numberBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>PDFSetupDialog</receiver>
   <slot>enableNumbers(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rowSizeBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>PDFSetupDialog</receiver>
   <slot>setRowSize(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>browsePath()</slot>
  <slot>setRowSize(int)</slot>
  <slot>setHMarginRatio(int)</slot>
  <slot>setVMarginRatio(int)</slot>
  <slot>setHSpacingRatio(int)</slot>
  <slot>setVSpacingRatio(int)</slot>
  <slot>enableNumbers(bool)</slot>
 </slots>
</ui>
